.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
         exclude: conda/meta.yaml
  
   - repo: https://github.com/psf/black
-    rev: 22.12.0
+    rev: 23.1.0
```

Comment:
Please also update `pre-commit-hooks` to `v4.4.0` and isort to `5.12.0`.

Reply:
Done in the commit: [ad8e2e3](https://github.com/lsst-ts/ts_m2gui/pull/34/commits/ad8e2e32b04894e7bf312394b459378a4110f482). Thanks!